subsys/net/l2_wifi_if_conn/Kconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig L2_WIFI_CONNECTIVITY
+	bool "L2 Wi-Fi Connectivity"
+	depends on NET_CONNECTION_MANAGER
+
+if L2_WIFI_CONNECTIVITY
+
+config L2_WIFI_CONN_WQ_STACK_SIZE
+	int "Wi-Fi connectivity work queue stack size"
```

Comment:
may be add a note as to why it should be high?

Reply:
Done